src/x-platform/index.js

```diff
@@ -1,26 +1,107 @@
+import 'html5-desktop-notifications'
+import {notificationsConfig} from './constants'
+import noop from 'lodash/utility/noop'
+
+
+// Electron brings own require to window
+let electron
+let remote
+if (window.require) {
+  electron = require('electron')
+  remote = require('remote')
+}
+
+const macGap = window.MacGap
```

Comment:
I did it this way because I had just 2 platforms to support, right now it became 3 and we don't know exactly when we will be able to remove macgap.